src/qml/pages/node/PeerDetails.qml

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2024 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import org.bitcoincore.qt 1.0
+import "../../controls"
+import "../../components"
+
+Page {
+    signal backClicked()
+
+    property PeerDetailsModel details
+
+    Connections {
+        target: details
+        function onDisconnected() {
+            root.backClicked()
+        }
+    }
+
+    id: root
+    background: null
+
+    header: NavigationBar2 {
+        leftItem: NavButton {
+            iconSource: "image://images/caret-left"
+            text: qsTr("Back")
+            onClicked: root.backClicked()
+        }
+        centerItem: Header {
+            headerBold: true
+            headerSize: 18
+            header: qsTr("Peer " + details.nodeId)
+        }
+    }
+
+    component PeerKeyValueRow: Row {
+        property string key: ""
+        property string value: ""
+        CoreText {
+            color: Theme.color.white;
+            text: key;
+            width: 125;
+            horizontalAlignment: Qt.AlignLeft;
+        }
+        CoreText {
+            text: value
+            horizontalAlignment: Qt.AlignLeft;
+        }
+    }
+
+    ScrollView {
+        id: scrollView
+        width: parent.width
+        height: parent.height
+        clip: true
+        contentWidth: width
+
+        Column {
```

Comment:
we need rules and way to wrap these values if for whatever reason they get too long: 

<img width="752" alt="Screenshot 2024-01-29 at 9 11 16 AM" src="https://github.com/bitcoin-core/gui-qml/assets/23396902/c1eede0e-2f77-4589-85ad-f4af0712aef5">

Reply:
i think a max width with an elipses should cover it. I'll see what that looks like.

Reply:
values now elide

Reply:
user might want to view all _Services_
maybe be good to put the missing ones in a tooltip?

Reply:
also things like onion address should be fully visible imo
 https://github.com/bitcoin-core/gui-qml/pull/375#pullrequestreview-1759750423